src/index.js

```diff
@@ -41,6 +41,15 @@ async function isWithinRange(forwardedFor, fastlyPublicIps, whitelistedIps = '')
     .some((myranges) => (range.isRange ? range.inRange(ip, myranges) : range === ip)));
 }
 
+async function getEmbedKind(domainList) {
+  const cpy = domainList;
```

Comment:
this doesn't do any good...the array is passed as reference. if you only want to avoid eslint errors, i'd rather disable the rule.